data/net/minecraft/world/level/chunk/storage/RegionFileVersion.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/world/level/chunk/storage/RegionFileVersion net/minecraft/world/level/chunk/storage/RegionFileVersion
+	COMMENT A decorator for input and output streams used to read and write the chunk data from region files..
```

Comment:
The comment should also describe that the class represents the known versions (compression methods) of the region file format, possibly.